src/mlpack/methods/adaboost/adaboost_main.cpp

```diff
@@ -155,10 +166,12 @@ static void mlpackMain()
   if (CLI::HasParam("input_model"))
     RequireAtLeastOnePassed({ "test" }, false, "no task will be performed");
 
-  RequireAtLeastOnePassed({ "output_model", "output" }, false,
+  RequireAtLeastOnePassed({ "output_model", "output", "predictions" }, false,
       "no results will be saved");
 
+  // "output" can be removed in mlpack 4
   ReportIgnoredParam({{ "test", false }}, "output");
```

Comment:
Since `output` is deprecated, I don't think we need this check anymore.

Reply:
Removed